lib/gpio/include/QEI.h

```diff
@@ -244,4 +231,10 @@ class QEI {
   volatile float movingAvgVelocity_PulsesPerSec_;
 
   float movingAvgSmoothingParam_;  // 0.1: minimal smoothing, 0.9: strong smoothing
+
+ protected:
+  static constexpr uint8_t PREV_MASK = 0x1;
+  static constexpr uint8_t CURR_MASK = 0x2;
+  static constexpr uint8_t INVALID   = 0x3;
```

Comment:
is the INVALID variable also a mask? If so it should probably also be called that. Maybe some more description could help cause I'm not sure what its used for

Reply:
i didnt write this but this sounds like a fair concern. tbh i didnt touch stuff i dont understand

Reply:
maybe raise it in an issue?

Reply:
@FK3wong am i okay to resolve this thread?